app/controllers/form_question_controller.rb

```diff
@@ -3,28 +3,79 @@ class FormQuestionController < ApplicationController
   def show
     @form_type = params[:type]
```

Comment:
We should change that logic later. User controller passes form_type because this was before we implemented the form table.